runtime/lua/vim/fs.lua

```diff
@@ -1,5 +1,7 @@
 local M = {}
 
+local iswin = vim.loop.os_uname().sysname == 'Windows_NT'
```

Comment:
Don't create custom os detection please, these things tend to proliferate.

```
local iswin = is_os('win')
```

is fine.

Reply:
Do we have `is_os` outside of the test suite?

Reply:
I can't read. I missed that this was outside of test. Ignore.